tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -118,49 +119,159 @@ spec:
           value: $(params.umb_url)
         - name: signature_data_file
           value: "$(params.signature_data_file)"
+        - name: CA_BUNDLE
+          value: /etc/pki/tls/certs/ca-bundle.crt
       script: |
         #!/usr/bin/env /bin/bash
         set -x
-        
-        MAX_RETRIES=3
-        RETRY_DELAY=5 # Initial delay
-        
-        set +x
-        # This helps with Shellcheck warning
-        echo "${UmbCert:?}" > /tmp/crt
-        echo "${UmbKey:?}" > /tmp/key
+        cat <<EOF > /tmp/pubtools-sign-config.yaml
+        msg_signer:
+          messaging_brokers:
+            - "${umb_url}"
+          messaging_cert_key: /tmp/umb.pem
+          messaging_ca_cert: ${CA_BUNDLE}
+          topic_send_to: ${umb_publish_topic}
+          topic_listen_to: queue://Consumer.${requester}.{task_id}.${umb_listen_topic}.{task_id}
+          environment: prod
+          service: ${umb_client_name}
+          timeout: 7
+          retries: 1
+          send_retries: 2
+          message_id_key: request_id
+          log_level: debug
+        EOF
+
+    - name: request-signature
+      image: "$(params.pipeline_image)"
+      workingDir: "$(workspaces.data.path)"
+      env:
+        - name: UmbCert
```

Comment:
Nit picking but.. could this and UmbKey use the same format as other parameters use in the same file to be more consistent? In this case lower case separating words with underscores.

Reply:
fixed